testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewControllerTest.m

```diff
@@ -60,4 +60,58 @@ - (void)testFirstFrameCallback {
   [self waitForExpectationsWithTimeout:30.0 handler:nil];
 }
 
+- (void)testDrawLayer {
+  XCTestExpectation* firstFrameRendered = [self expectationWithDescription:@"firstFrameRendered"];
+  XCTestExpectation* imageRendered = [self expectationWithDescription:@"imageRendered"];
+
+  FlutterEngine* engine = [[FlutterEngine alloc] initWithName:@"test" project:nil];
+  [engine runWithEntrypoint:nil];
+  [engine.binaryMessenger
+      setMessageHandlerOnChannel:@"waiting_for_status"
+            binaryMessageHandler:^(NSData* _Nullable message, FlutterBinaryReply _Nonnull reply) {
+              FlutterMethodChannel* channel = [FlutterMethodChannel
+                  methodChannelWithName:@"driver"
+                        binaryMessenger:engine.binaryMessenger
+                                  codec:[FlutterJSONMethodCodec sharedInstance]];
+              [channel invokeMethod:@"set_scenario" arguments:@{@"name" : @"solid_blue"}];
+            }];
+
+  self.flutterViewController = [[FlutterViewController alloc] initWithEngine:engine
+                                                                     nibName:nil
+                                                                      bundle:nil];
+
+  XCTAssertFalse(self.flutterViewController.isDisplayingFlutterUI);
+
+  [self.flutterViewController setFlutterViewDidRenderCallback:^{
+    [firstFrameRendered fulfill];
+  }];
+
+  AppDelegate* appDelegate = (AppDelegate*)UIApplication.sharedApplication.delegate;
+  UIViewController* rootVC = appDelegate.window.rootViewController;
+  [rootVC presentViewController:self.flutterViewController animated:NO completion:nil];
+
+  CGColorSpaceRef color_space = CGColorSpaceCreateDeviceRGB();
```

Comment:
I believe this is leaking, there should be a CGColorSpaceRelease.

Reply:
Adding this in a follow up, and for the CGContextRelease